lib/cookie.c

```diff
@@ -368,6 +368,18 @@ static void strstore(char **str, const char *newstr, size_t len)
   *str = Curl_memdup0(newstr, len);
 }
 
+static time_t time_now(void)
```

Comment:
Won't it be a bit odd if cookies run on a different clock from the rest of curl during a debugbuild testrun? Shouldn't this be a common function and all `time(NULL);` calls be replaced with `time_now();`?

Reply:
Thinking about this further: I believe it is also problematic to change all times because some timers actually still need to run proper and for debugging some timers are better set fixed...